silver_braintree/payment_processors.py

```diff
@@ -370,9 +373,50 @@ def execute_transaction(self, transaction):
 
         return self._charge_transaction(transaction)
 
+    def find_lost_transaction_id(self, transaction):
```

Comment:
I'll just rename it to `recover_lost_transaction_id`. I don't think there's any point in splitting it as the parts don't have any reusability value.